README.md

```diff
@@ -15,35 +15,52 @@ A PHP7 implementation of the [GraphQL specification](http://facebook.github.io/g
 
 When we started developing this project there were two GraphQL implementations available for PHP developers; one from 
 [Webonyx](https://github.com/webonyx/graphql-php/) and one from [Youshido](https://github.com/youshido/graphql/). 
-The one from Webonyx is a direct port of the reference implementation and it includes a lot of things that we think 
-should be provided as separate libraries. The other implementation from Youshido has some neat ideas, but unfortunately 
-it does not fulfill the GraphQL specification. After doing some research, we concluded that neither of these
-implementations met our needs, so we decided to start coding and see what we would come up with.
+The one from Webonyx is a direct port of the reference implementation and it does not fully use modern PHP. The other 
+implementation from Youshido has some neat ideas, but unfortunately it does not fulfill the GraphQL specification, 
+e.g. it does not come with any query validation. After some research, we concluded that neither of these implementations 
```

Comment:
Actually I noticed that it does indeed seem to do some validation, e.g. "Fragment X is not used"

Reply:
I guess we could say "proper query validation".

Reply:
The reason I wrote this is because I tried at least 5 different validation cases on one of our APIs and they all passed even though they shouldn't.

Reply:
Fixed.